invokeai/app/cli_app.py

```diff
@@ -162,8 +164,8 @@ def invoke_cli():
 
     while True:
         try:
-            cmd_input = input("> ")
-        except KeyboardInterrupt:
+            cmd_input = input(f"{model_manager.current_model or '(no model)'}> ")
```

Comment:
Model can be specified as part of the node with `--model`. There's a command to view and set defaults, and that's where the "default" model should be managed (i.e. it shouldn't be part of the command prompt).

Reply:
Ah. I think we probably need to set a default at CLI startup so it'll always be populated if not provided.